iaso/api/payments/serializers.py

```diff
@@ -76,28 +103,50 @@ class Meta:
 class PaymentLotSerializer(serializers.ModelSerializer):
     payments = serializers.SerializerMethodField()
     task = NestedTaskSerializer(read_only=True)
+    can_see_change_requests = serializers.SerializerMethodField()
 
     class Meta:
         model = PaymentLot
-        fields = ["id", "name", "status", "created_at", "created_by", "payments", "comment", "task"]
+        fields = [
+            "id",
+            "name",
+            "status",
+            "created_at",
+            "created_by",
+            "payments",
+            "comment",
+            "task",
+            "can_see_change_requests",
+        ]
         read_only_fields = ["id", "created_at"]
 
     pagination_class = PaymentPagination
     created_by = UserNestedSerializer()
     created_at = TimestampField(read_only=True)
 
+    def get_can_see_change_requests(self, obj):
+        user = self.context.get("request").user
+        if user.is_superuser:
+            return True
+        user_org_units = sorted(list(user.iaso_profile.org_units.all().values_list("id")))
```

Comment:
I think that instead of accessing org units linked directly to the user, it should be the hierarchy of the org units